.gitattributes

```diff
@@ -0,0 +1,3 @@
+static/header_short.png filter=lfs diff=lfs merge=lfs -text
```

Comment:
Please track all PNG files with `static/*.png`

Reply:
I added new image file type (`svg`) instead of `png`s